src/structures/GuildMember.js

```diff
@@ -284,6 +304,18 @@ class GuildMember extends Base {
     return this.edit({ nick }, reason);
   }
 
+  
+  /**
+   * Disables or enables communication for this member.
+   * @param {?number | Date} seconds The duration in seconds or a date in the future until the member should have
```

Comment:
The documentation doesn't render the `?` properly otherwise
```suggestion
   * @param {?(number|Date)} seconds The duration in seconds or a date in the future until the member should have
```